ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/Abstract/ITaskService.cs

```diff
@@ -14,5 +14,13 @@ public interface ITaskService
         Task<TaskDTO> CreateAsync(CreateTaskDTO task);
 
         Task<List<TaskDTO>> GetTasksOnDateAsync(DateTime date);
```

Comment:
remove word Task/Tasks from names of methods, it's duplication of service name (TaskService)